docker-compose.yml

```diff
@@ -377,6 +377,23 @@ services:
     command: daemon --docker -f label=com.docker.compose.project=${COMPOSE_PROJECT_NAME}
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock:ro
+  mailhog:
+    image: mailhog/mailhog:latest
+    container_name: mailhog
+    profiles:
+      - email
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8025"]
```

Comment:
This healthcheck fails for me because the container does not have curl in its PATH.

Reply:
Fixed!

I changed it to use wget instead. I checked and it's available

![image](https://github.com/stackernews/stacker.news/assets/241271/34a41e9a-c908-4c02-a69e-fb7c56ef22ef)